migration-roadmap/infield-button.md

```diff
@@ -0,0 +1,297 @@
+- `.spectrum-InfieldButton:disabled`
+- `.spectrum-InfieldButton:focus-visible`
+- `.spectrum-InfieldButton:hover`
+- `.spectrum-InfieldButton:not(:disabled):active`
+- `.spectrum-InfieldButton:not(:disabled):focus-visible`
+- `.spectrum-InfieldButton:not(:disabled):hover`
+
+</details>
+
+<details>
+<summary>Passthroughs</summary>
+
+None found for this component.
+
+</details>
+
+<details>
+<summary>Modifiers</summary>
+
+- `--mod-infield-button-background-color`
+- `--mod-infield-button-background-color-disabled`
+- `--mod-infield-button-background-color-down`
+- `--mod-infield-button-background-color-down-disabled`
+- `--mod-infield-button-background-color-down-quiet`
+- `--mod-infield-button-background-color-hover`
+- `--mod-infield-button-background-color-hover-disabled`
+- `--mod-infield-button-background-color-hover-quiet`
+- `--mod-infield-button-background-color-quiet`
+- `--mod-infield-button-background-color-quiet-disabled`
+- `--mod-infield-button-border-radius`
+- `--mod-infield-button-edge-to-fill`
+- `--mod-infield-button-field-edge-to-icon`
+- `--mod-infield-button-fill-justify-content`
+- `--mod-infield-button-fill-padding`
+- `--mod-infield-button-height`
+- `--mod-infield-button-icon-color`
+- `--mod-infield-button-icon-color-disabled`
+- `--mod-infield-button-icon-color-down`
+- `--mod-infield-button-icon-color-down-disabled`
+- `--mod-infield-button-icon-color-hover`
+- `--mod-infield-button-icon-color-hover-disabled`
+- `--mod-infield-button-side-edge-to-fill`
+- `--mod-infield-button-width`
+
+</details>
+
+### SWC
+
+<details>
+<summary>Attributes</summary>
+
+- `active` (Boolean) - Whether the infield button is in an active state
+- `block` (String) - Vertical stack position: 'start' or 'end'
+- `disabled` (Boolean) - Disable this control. It will not receive focus or events
+- `download` (String) - Causes the browser to treat the linked URL as a download
+- `href` (String) - The URL that the hyperlink points to
+- `inline` (String) - Horizontal group position: 'start' or 'end'
+- `label` (String) - An accessible label that describes the component. It will be applied to aria-label, but not visually rendered
+- `quiet` (Boolean) - Whether the button is in quiet variant
+- `referrerpolicy` (String) - How much of the referrer to send when following the link
+- `rel` (String) - The relationship of the linked URL as space-separated link types
+- `tabIndex` (Number) - The tab index to apply to this control
+- `target` (String) - Where to display the linked URL, as the name for a browsing context
+- `type` (String) - The default behavior of the button: 'button', 'submit', or 'reset'
+
+</details>
+
+<details>
+<summary>Slots</summary>
+
+- Default slot - Icon content for the button
+
+</details>
+
+## Comparison
+
+### DOM Structure changes
+
+<details>
+<summary>Spectrum Web Components:</summary>
+
+```html
+<button class="spectrum-InfieldButton">
+    <div class="fill">
+        <slot></slot>
+    </div>
+</button>
```

Comment:
Correct me if I'm wrong, but is there a `button` tag at all in this component? I don't think I see it in the render(), but `sp-infield-button` also extends `ButtonBase`, so maybe it's...implied somehow? I'm not familiar enough with this component or ButtonBase.

Reply:
@cdransf These are great questions and I don't have an answer just glancing at the resources. Is it okay to leave this as an open topic in the doc or do we need a definitive answer?